src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/WidgetPageObject.java

```diff
@@ -64,6 +68,9 @@ public WidgetPageObject createMultiple() {
     return this;
   }
 
+  /**
+   * Create incorrect widget tag on the tested article
```

Comment:
Now this comment just duplicates what's in the code, but putting more info, as you mentioned, would make this pretty useful — go for it! :+1: :sunglasses: